### PR TITLE
Bump all JS dev dependencies to latest

### PR DESCRIPTION
**Pull request by jamescrowley**
Builds on #1519 (so merge that first) - including those commits to avoid conflicts in the lock file.

### CODE REVIEW

**Comment by jamescrowley** [comment]
eslint has significant breaking changes, so I'll do this seperately

File: .github/dependabot.yml [hunk 1]
```diff
@@ -34,3 +34,8 @@ updates:
     # Always increase the minimum version requirement to match the new version.
     # https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#versioning-strategy
     versioning-strategy: "increase"
+    groups:
+      nivo-deps:
```
Comment (fhenrich33) [inline .github/dependabot.yml:38]: Is there a doc where I can learn more about this dependabot feature?

**Review by fhenrich33 — APPROVED** [review]

File: .github/dependabot.yml (hunk 1 above)
Comment (jamescrowley) [reply .github/dependabot.yml]: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#groups :)

**Comment by jamescrowley** [comment]
Closing this for now - will follow up with more limited set of dev dependencies

**Pull request closed without merging**